apps/builddao/widget/components/modals/propose/Transfer.jsx

```diff
@@ -1,13 +1,22 @@
 const { Button } = VM.require("${config_account}/widget/components") || {
   Button: () => <></>,
 };
+const { ProposalVisibilityInfoModal } = VM.require(
+  "${config_account}/widget/components.modals.propose.ProposalVisibilityInfoModal",
+) || {
+  ProposalVisibilityInfoModal: () => <></>,
+};
 const DaoSDK = VM.require("sdks.near/widget/SDKs.Sputnik.DaoSDK") || (() => {});
 
 const [recipient, setRecipient] = useState("");
 const [token, setToken] = useState("");
 const [amount, setAmount] = useState(0);
 const [description, setDescription] = useState("");
 const [validatedAddresss, setValidatedAddress] = useState(true);
+const [isInfoModalActive, setInfoModalActive] = useState(false);
+const [copied, setCopied] = useState(false);
+const url =
+  "https://www.nearbuilders.org/buildhub.near/widget/app?page=feed&tab=proposals";
```

Comment:
I see this url in quite a few cases; in each of them we want it to be the correct url according to environment. For this, we use [bos-workspace aliases](https://github.com/NEARBuilders/bos-workspace?tab=readme-ov-file#aliases).

I think this should be:
`${alias_gateway_url}/${config_account}/widget/app?page=feed&tab=proposals`

Reply:
Sure .Implemented this.